### PR TITLE
add application button on home page for super user and added status field in application details

### PR DESCRIPTION
**Pull request by vinayak-trivedi**
Date: 6/01/2024

## Description

This PR adds application button on home page for super user and also addds status field in application details page.

**Documentation Updated?**

-   [ ] Yes
-   [x] No

_If your feature adds a new API, then documentation related to the feature like API-Contracts, and Data-models  must be updated._

**Breaking Changes**

-   [ ] Yes
-   [x] No

_If your feature introduces breaking changes or if something is missing, please mention the related issue tickets._

**Development Tested?**

-   [x] Yes
-   [ ] No

_Confirm whether the changes have been tested locally during development._

**Tested in Staging?**

-   [ ] Yes
-   [x] No

_Indicate whether the changes have been tested in the staging environment._

**Under Feature Flag**

-   [ ] Yes
-   [x] No

_Specify if the changes are currently under a feature flag._

**Database Changes**

-   [ ] Yes
-   [x] No

image

### CODE REVIEW

**Review by anishpawaskar — CHANGES_REQUESTED** [review]
Please write test

**Comment by satyam73** [comment]
Please add a screenshot of the UI change

**Review by anishpawaskar — APPROVED** [review]

File: index.html [hunk 1]
```diff
@@ -149,6 +149,13 @@
       </a>
       <a class="action-button" href="/groups/index.html"> Discord Groups </a>
       <a class="action-button" href="/standup/index.html"> Standup Updates</a>
+      <a
+        id="application-button"
+        href="/applications"
```
Comment (vikhyat187) [inline index.html:154]: Hi @vinayak-trivedi can we have a better naming here, I would not have been able to understand what application does, if this was viewed without any context by me.

Comment (vikhyat187) [reply index.html:154]: Discussed this, it takes to Applications page.

**Review by vikhyat187 — APPROVED** [review]

**Pull request merged**